src/tutorial.md

```diff
@@ -1,122 +1,106 @@
-:::
+This article is a tutorial on developing Denops plugins.
 
-どうも、最近 Rust と Deno にハマってるありすえです。
+## Environment
 
-今日は [vim-jp][] で開発を開始した [denops.vim][] の紹介と denops.vim を利用したプラグイン作成のチュートリアルを書きたいと思います。
+In this tutorial, we use the following software and version as of writing.
+* [denops.vim v1.0.0-beta.4](https://github.com/vim-denops/denops.vim/releases/tag/v1.0.0-beta.4) (2021-07-11)
+* [denops_std v1.0.0-beta.2](https://github.com/vim-denops/deno-denops-std/releases/tag/v1.0.0-beta.2) (2021-07-11)
 
 [vim-jp]: https://vim-jp.org/
 [denops.vim]: https://github.com/vim-denops/denops.vim
-
-# denops.vim とは
-
-denops.vim は JavaScript/TypeScript のランタイムである [Deno][] を利用して Vim/Neovim 双方で動作するプラグインを作るためのエコシステムです。以下のような特徴があります。
-
-- Vim / Neovim で同一コードを利用可能
-- Vim プラグインとしてインストールが可能
-- Vim script と比較してエンジンが爆速なのでゴリ押しが可能
-- ユーザーによるライブラリの依存管理が不要
-- プラグインが別プロセスとして動作するため Vim が固まりにくい
-- プラグイン毎にスレッドが分かれているため相互干渉が起こりにくい
-
 [deno]: https://deno.land/
 
-# 用語集
+## Glossary
 
-| 用語              | 意味                                                                    |
-| ----------------- | ----------------------------------------------------------------------- |
-| Denops            | Deno をランタイムとして利用した Vim/Neovim のプラグインエコシステムです |
-| Denops プラグイン | Denops を用いて書かれた Vim/Neovim 双方で動作するプラグインを表します   |
+| Term                  | Meaning                                                                     |
+| --------------------- | --------------------------------------------------------------------------- |
+| vim                   | Vim or NeoVim.                                                          |
+| vim plugin            | Vim plugin or NeoVim plugin.                                            |
+| [Deno][]              | A JavaScript and TypeScript runtime.                                        |
+| [Denops][denops.vim]  | An ecosystem for vim plugins based on Deno runtime.                         |
+| Denops plugin         | A vim plugin that works on both Vim and NeoVim and is written with Denops.  |
+| [denops.vim][]        | The Name of the vim plugin to introduce Denops into vim.                      |
 
-# Denops のインストール
+## Preparation
 
-Denops プラグインを利用するためには Denops 自体をインストールする必要があります。
-これは Denops プラグインを利用するだけのユーザーも行う必要があります。
+First of all, whichever you want to either use or develop Denops plugins, you have to install tools; [Deno][] and [Denops][denops.vim] in addition to your vim.
 
-## 0. Deno のインストール
+### Installing Deno
 
-https://deno.land/#installation を参考に Deno をインストールしてください。
-Getting Started の以下のコマンドを実行して結果が帰ってくれば成功です。
+Deno can be installed to follow the instructions in the [Deno document](https://deno.land/#installation).
+In addition, you can check if Deno has been installed successfully by [the command](https://deno.land/#getting-started):
 
 ```sh
 deno run https://deno.land/std/examples/welcome.ts
 ```
 
-なお、既にインストール済みであれば、以下のコマンドで最新版にアップデートしてください。
+If you have already installed Deno, update it to the latest version.
```

Comment:
Let use "upgrade" rather than "update" while the command below says "upgrade" instead.

Reply:
Corrected at af3096c.